server/report/report-remap.js

```diff
@@ -28,13 +40,11 @@ export default class {
     process.chdir(Conf.COVERAGE_APP_FOLDER);
 
     // Create output directory if not exists
-    let remapFolder = path.join(Conf.COVERAGE_EXPORT_FOLDER, 'remap');
-    let remapPath = path.join(Conf.COVERAGE_APP_FOLDER, remapFolder);
-    ReportCommon.checkDirectory(remapPath);
+    ReportCommon.checkDirectory(this.remapPath);
 
-    let addFile = (filename) => path.join(remapFolder, filename);
+    let addFile = (filename) => path.join(this.remapFolder, filename);
```

Comment:
maybe create a function `getFilePath` in the class `ReportRemap`? this.remapFolder would not be passed to that function in that case